core-java-modules/core-java-lang-operators-2/src/test/java/com/baeldung/operators/ArithmeticUnitTest.java

```diff
@@ -0,0 +1,52 @@
+package com.baeldung.operators;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class ArithmeticUnitTest {
+
+    @Test
+    void whenUseTheOperatorArithmeticAdditionToConcatenate() {
+        String sentence = "Hello world!";
+        String wordsConcatenated = "Hello " + "world" + "!";
+        assertEquals(sentence, wordsConcatenated);
+
+    }
+
+    @Test
+    void whenUseTheOperatorArithmeticAdditionToOperation() {
+        int resultSumPositive = 10 + 5;
+        assertEquals(resultSumPositive, 15);
+
```

Comment:
I meant this extra space before the closing curly brace.

Reply:
Done.